static/js/settings_emoji.js

```diff
@@ -183,6 +183,16 @@ exports.set_up = function () {
                 emoji[obj.name] = obj.value;
             }
 
+            if (emoji.name === "") {
```

Comment:
Probably should do a `trim()` before this check, so " " doesn't count.

Reply:
Thanks for reviewing. You mean I should do ```emoji.name.trim() === ""``` ?